store/mockstore/unistore/cophandler/mpp_exec.go

```diff
@@ -231,6 +236,12 @@ func (e *exchRecvExec) open() error {
 }
 
 func (e *exchRecvExec) next() (*chunk.Chunk, error) {
+	if !e.inited {
```

Comment:
Why move this part?

Reply:
I'm trying to pass all the errors (including injected error) to ErrChannel, which can not be implemented in the "open" function. The error is returned by "next" function and is captured in the "exchSender", then be passed to the next task, util the error approches client.